pkg/provider/cloud/vsphere/folder_test.go

```diff
@@ -64,30 +64,26 @@ func TestProvider_GetVMFolders(t *testing.T) {
 		name            string
 		dc              *kubermaticv1.DatacenterSpecVSphere
 		expectedFolders sets.String
-		// If we check for folders on the root we might have other tests creating folders, so we have to skip that check
-		skipAdditionalFoldersCheck bool
```

Comment:
There was a folder named after a cluster in our vCenter. Assuming that can always happen, I decided to make all folder tests to only check for a subset, not equality.